Mathlib/Algebra/Polynomial/Taylor.lean

```diff
@@ -167,7 +167,7 @@ section CommRing
 variable {R : Type*} [CommRing R] (r : R) (f : R[X])
 
 /-- `Polynomial.taylor` as an `AlgEquiv` for commutative rings. -/
-noncomputable def taylorEquiv (r : R) : R[X] ≃ₐ[R] R[X] where
+@[simps!] noncomputable def taylorEquiv (r : R) : R[X] ≃ₐ[R] R[X] where
```

Comment:
What's the point of this `simps` if it only generates `symm_apply` which is already covered by `taylorEquiv_symm`?

Reply:
removed